src/Xamarin.Android.Build.Tasks/Tasks/AppBundleBaseZip.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.IO;
+using Xamarin.Tools.Zip;
+
+namespace Xamarin.Android.Tasks
+{
+	public class AppBundleBaseZip : BuildApk
```

Comment:
A more semantically meaningful name would be useful; there's no *verb* here, just a noun.  `<BuildAppBundle/>` might be more meaningful when reading the `.targets` file, or `<ConvertApkToAppBundle/>`.

Reply:
Yeah `<BuildBaseAppBundle/>`, because it is not actually an app bundle yet... It's a "base zip" file that is input to `bundletool`.